packages/language-server/api.d.ts

```diff
@@ -25,6 +25,13 @@ export type ServerInitializationOptions = {
   logLevel?: LogLevel;
 };
 
+export function getNodeName(
+  text: string,
```

Comment:
with so many arguments, some of which are optional you should probably use a config object in this case.